test/multiarch_utils.go

```diff
@@ -71,14 +72,24 @@ func initImageNames() map[int]string {
 			kanikoImage:    "gcr.io/kaniko-project/executor:s390x-9ed158c1f63a059cde4fd5f8b95af51d452d9aa7",
 			dockerizeImage: "ibmcom/dockerize-s390x",
 		}
-	}
-	return map[int]string{
-		busyboxImage:   "busybox@sha256:895ab622e92e18d6b461d671081757af7dbaa3b00e3e28e12505af7817f73649",
-		registryImage:  "registry",
-		kubectlImage:   "lachlanevenson/k8s-kubectl",
-		helmImage:      "alpine/helm:3.1.2",
-		kanikoImage:    "gcr.io/kaniko-project/executor:v1.3.0",
-		dockerizeImage: "jwilder/dockerize",
+	case "ppc64le":
+		return map[int]string{
+			busyboxImage:   "busybox@sha256:4f47c01fa91355af2865ac10fef5bf6ec9c7f42ad2321377c21e844427972977",
+			registryImage:  "ppc64le/registry:2",
+			kubectlImage:   "ibmcom/kubectl:v1.13.9",
+			helmImage:      "ibmcom/helm-ppc64le:latest",
+			kanikoImage:    "ibmcom/kaniko-project-executor-ppc64le:v0.17.1",
```

Comment:
Is there any possibility of having ppc64le support in the official kaniko images? They recently started publishing multiarch images, so it shouldn't be too difficult to add unless I'm missing something